lib/data.dart

```diff
@@ -7,6 +7,25 @@ class Year {
   String id;
 
   Year(this.name, this.id);
+
+  @override
+  String toString() {
+    return "Year(id: $id, name: $name)";
+  }
+
+  @override
+  bool operator ==(Object other) {
+    try {
+      // ignore: test_types_in_equals
+      return this.id == (other as Year).id;
+    } on TypeError {
+      return false;
+    }
+  }
+
+  @override
+  // TODO: implement hashCode
+  int get hashCode => super.hashCode;
```

Comment:
`contains` works weird without overwriting `==` and when you do it, Android Studio annoys to make hashCode. I added hashcode to avoid warnings and actually while overwriting comparison method, hashcode should have been overwritten too.

Reply:
Why u need contains? Class instances don't have hidden fields, u could reference them just as  
`Year year = new Year('year',1337);
`assert(year.id == 1337)`